bin/si-mcp-server/src/tools/componentDelete.ts

```diff
@@ -0,0 +1,76 @@
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { z } from "zod";
+import { ComponentsApi } from "@systeminit/api-client";
+import { apiConfig, WORKSPACE_ID } from "../si_client.ts";
+import {
+  errorResponse,
+  generateDescription,
+  successResponse,
+} from "./commonBehavior.ts";
+
+const name = "component-delete";
+const title = "Delete a component";
+const description =
+  `<description>Delete a component for a given componentId. Returns the deletion status on successful deletion. On failure, returns error details.</description><usage>Use this tool to delete a component in a change set. The component will be marked for deletion and removed from the workspace.</usage>`;
```

Comment:
You should add to the description that this will cause any resource to also be deleted. That's setting the stage a little bit for adding an 'erase' endpoint that keeps the resource around.

This can merge without that change tho.